lib/api/index.js

```diff
@@ -359,12 +359,26 @@ app.post('/:installationId/migrate/:undo?', [
 
   try {
     if (req.params.undo) {
-      await jiraClient.devinfo.migration.undo();
+      /**
+       * Only call github/migrationUndo in prod. Undo will only be called in Jira if
+       * GITHUB_CONNECT_APP_IDENTIFIER is equal to com.github.integration.production
+       */
+      if (process.env.NODE_ENV === 'production') {
```

Comment:
Having logic to see if we're in production in several places sprinkled through the code is not great.  How about we create our own `jiraClient` wrapper so we can centralize that logic into a single file?